README.md

```diff
@@ -473,6 +473,14 @@ All statements above will return false if used with `===`
 
 <p align="right"><a href="#table-of-contents">⬆ Return to top</a></p>
 
+## ![✔] 4.9 Refactor regularly using static analysis tools
+
+**TL;DR:** Using static analysis tools helps give objective ways to improve code quality and keep your code maintainable. Keeping out code smells and bloat will make maintenance easier and provides better readability to your code. Try tools like [Code Climate](https://codeclimate.com/) or [Codebeat](https://codebeat.co/) to automate finding code to improve.
```

Comment:
I'm not sure but maybe for the text flow you could change 'helps give objective ways to...' to either 'helps to give objective ways to...' or rather 'helps by giving objective ways to...'

Reply:
Ah, 'helps by giving objective ways to...' does sound better!